### PR TITLE
Support for multiple terms in a grammar.

### PR DESCRIPTION
**Pull request by balu**
This pull request adds support for multiple terms (variables) in a grammar. The documentation is also updated. I removed fontspec from documentation as it is not supported by latex.

Balagopal.

### CODE REVIEW

File: simplebnf-doc.tex [hunk 1]
```diff
@@ -1,82 +1,81 @@
...
+  \footnote{This file describes v0.2.0.}}
 \author{Jay Lee\footnote{E-mail: %
   \href{mailto:jaeho.lee@snu.ac.kr}{\texttt{jaeho.lee@snu.ac.kr}}}}
-\date{2019/12/23}
+\date{2020/09/01}
 
 \begin{document}
 \maketitle
 
+This package provides a simple way to typeset grammars written in BNF form.
```
Comment (Zeta611) [inline simplebnf-doc.tex]: _form_ in _BNF form_ is redundant, as BNF already is an acronym for Backus-Naur Form. IMHO, the first occurrence of the term BNF should be preceded by the full name, e.g., _Backus-Naur form(BNF)_.

File: simplebnf-doc.tex [hunk 2]
```diff
@@ -1,82 +1,81 @@
...
-to define a grammar.
-The text inside the environment should be formatted as:
-\begin{equation*}
-  \bnfexpr{term} \Coloneqq \langle\textit{keypairs}\rangle
-\end{equation*}
-where each pair of \textit{keypairs} represents an alternative syntactic form
-of the \textit{term} and its annotation, delimited with a colon(\verb/:/).
-If you don't need annotations, simply omit the colons and annotations
-altogether.
+
```
Comment (Zeta611) [inline simplebnf-doc.tex]: This empty line creates a new paragraph and should be removed; the previous _presentcommand_ block is logically connected to the following sentence. The same applies to line 55, 64, and 69 as well.

**Review by Zeta611 — CHANGES_REQUESTED** [review]
Thank you so much the contribution! I added some feedback to the documentation. Besides that, ILGTM.

File: simplebnf-doc.tex (hunk 1 above)
Comment (balu) [reply simplebnf-doc.tex]: I've amended the commit with all requested changes.

Comment (Zeta611) [reply simplebnf-doc.tex]: Thanks a lot!

**Pull request merged**